static/app/components/button.tsx

```diff
@@ -193,6 +195,16 @@ type LinkButtonProps =
   | HrefLinkButtonPropsWithChildren
   | HrefLinkButtonPropsWithAriaLabel;
 
+/**
+ * Default sizes to use for SVGIcon
+ */
```

Comment:
Maybe this is like, "map from button sizes to reasonable icon sizes" because the two enums are not the same.

OR we make them have the same keys so the mapping isn't needed idk